client/packages/common/src/ui/components/buttons/IconButton/IconButton.tsx

```diff
@@ -8,6 +8,7 @@ interface ButtonProps {
   label: string;
   width?: string;
   height?: string;
+  color?: 'primary' | 'secondary' | undefined;
```

Comment:
You'll notice this isn't used in this PR -- I'm using it for the Arrays to create the "Add item" button -- in next PR